app/forms/cancellation/CancellationChangeReason.scala

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright 2019 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package forms.cancellation
+
+object CancellationChangeReason extends Enumeration {
```

Comment:
Why you use Enumeration here? What do you think about sealed trait?

Reply:
I wanted to use the Enum because I could then serialise and deserialise from the request much easiser. In the end I did not end up doing that.
https://stackoverflow.com/questions/1898932/case-objects-vs-enumerations-in-scala
These is this interesting page about it. 
If you think we should use the sealed trait then I am happy to change, if not I believe I am not introducing any disruption to the code base ;)

Reply:
As you see I approved. I think sealed trait is better approach but more difficult or time consuming.

Enumeration is okay, just personally don't like it, I don't want to block it because of it :) So it's up to you

Reply:
Ok! Thank you! :)